classes/test.php

```diff
@@ -396,6 +396,16 @@ public function setAssertionManager(test\assertion\manager $assertionManager = n
 			->setPropertyHandler('exception', function() { return asserters\exception::getLastValue(); })
 		;
 
+		// Around testedInstance
+		$this->assertionManager
+			->setHandler('callStaticOnTestedClass', function() use ($test) {
```

Comment:
It's an indentation using tab, like the rest of the file.
Do you want to change the indentation just for this addition ?